### DISCUSSION TITLE
Possible to add CUDA support to the docker image?

### DISCUSSION THREAD

**Discussion by feltz916** [absadiki/subsai#102] [General]
its a long an tedious process :(

**Comment by absadiki** [comment]
I think I've used the cuda-pytorch docker image to build, so cuda should be supported. Doesn't work on your end ?

**Comment by feltz916** [comment]
I cannot get the GPU option to be available. I've tried docker desktop for windows with WSL2, and uninstalled that and tried docker integrated into my WSL ubutnu 20.04. In both cases, only None and CPU are options.

![image](https://github.com/abdeladim-s/subsai/assets/7417818/56311de1-785d-4e0c-bd30-69cad021fbc7)

**Reply by feltz916** [reply]
![image](https://github.com/abdeladim-s/subsai/assets/7417818/5787cdbf-0bbc-4e68-b8b8-05a3343945dc)

**Comment by absadiki** [comment]
@feltz916, I tried to add the gpu to the docker file. 
Pull the changes and give it a try now ?

**Comment by feltz916** [comment]
Thanks! Its now showing up.
